source/whats-new.txt

```diff
@@ -12,3 +12,250 @@ What's New
+  in an aggregate expression context. ``Let`` must be a document that maps 
+  parameter names to values that are constant or closed expressions without 
+  references to document fields. MongoDB v5.0 or later is required.
+
+- Added new constructor functions that create ``Cursor`` and ``SingleResult`` 
+  instances from marshalable and non-nil BSON documents.
+
+
+.. _version-1.8:
+
+What's New in 1.8
+-----------------
+
+New features of the 1.8 Go driver release include: 
+
+- Added full support for MongoDB 5.1.
+
+- Support for KMIP (Key Management Interoperability Protocol) as a KMS 
+  provider for client side encryption. 
+
+- Redesigned driver connection pool for low operation ``Context`` timeouts and 
+  to reduced connection churn. Behavior changes include: 
+
+  - New connection creation times out at ``connectTimeoutMS``.
+
+  - At most, two new connections can be established at the same time.
+
+- Removed oppressive and unncessarily gendered language in the Go driver 
+  documentation, code, tests, and spec tests. 
+
+
+.. _version-1.7:
+
+What's New in 1.7
+-----------------
+
+.. important:: Upgrade to Version 1.7.2 or Higher 
+
+  The 1.7.2 Go driver contains a bug fix for a data race that can occur between 
+  creating and checking out connections when ``minPoolSize > 0``. 
+
+New features of the 1.7 Go driver release include:
+
+- Added full support for MongoDB 5.0.
+
+- Support for the :readconcern:`"snapshot"` read concern outside of 
+  multi-document transactions for certain read operations.
+
+- Improved ``WriteException`` and ``BulkWriteException`` error messages for
+  schema validation via the ``WriteError.Details`` field.
+
+
+.. _version-1.6:
+
+What's New in 1.6
+-----------------
+
+.. important:: Upgrade to Version 1.6.2 or Higher 
+
+  The 1.6.2 Go driver contains a bug fix for a data race that can occur between 
+  creating and checking out connections when ``minPoolSize > 0``. 
+
+New features of the 1.6 Go driver release include:
+
+- Added support for the MongoDB Stable API. For more information, see the 
+  :ref:`Stable API Guide <golang-stable-api>`.
+
+- Added support for connections to any MongoDB service that runs behind a load 
+  balancer. 
+  
+- Added support for creating time series collections. For more information, see 
+  the :ref:`Time Series Collections Guide <golang-time-series>`.
+
+- Added the ``Let`` option on aggregates.
+
+
+.. _version-1.5:
+
+What's New in 1.5
+-----------------
+
+New features of the 1.5 Go driver release include: 
+
+- Added support for Azure and GCP key management services with client-side field 
+  level encryption.
+
+- Added a new errors API to detect duplicate key errors, timeouts, and network 
+  errors.
+
+- Added server monitoring to monitor changes on a MongoDB deployment. 
+
+- Added errors to prevent unexpected behavior on maps that contain multiple 
+  keys being used as a hint option, sort option, or for index creation.
```

Comment:
Issue: This suggests that "as a" should be applied to each element in the list: "...being used as a hint option, [as a] sort option, or [as a] for index creation." Of course, the last one doesn't make sense, since the multiple keys are being used _for_ something instead of _as_ something.

Suggestion: 
```suggestion
  keys being used as a hint option, as a sort option, or for index creation.
```

Reply:
Thanks for catching that! Fixed.